README.md

```diff
@@ -32,6 +33,16 @@ See this guide to [install the Open Raadsinformatie API](https://github.com/open
 We use [Sphinx](http://sphinx-doc.org/) to create the documentation. The source files are included in this repo under 
 the ``docs`` directory.  
 
+## Supported Sources
+
+Data extraction support is available for the following source systems:
+
+- [GO GemeenteOplossingen](https://www.gemeenteoplossingen.nl/)
+- [GreenValley](https://www.greenvalley.nl/)
+- [Notubiz](https://notubiz.nl/)
+- [Parlaeus](https://parlaeus.nl/)
```

Comment:
We don't support Parlaeus yet. The source file is there but it hasn't been implemented yet, so it's probably best to remove this for now.

Reply:
Trying to add commit, but not yet ready.